python/src/transforms.cpp

```diff
@@ -961,12 +960,22 @@ void init_transforms(nb::module_& m) {
         Globally enable compilation. This will override the environment
         variable ``MLX_DISABLE_COMPILE`` if set.
       )pbdoc");
+  m.def(
+      "clear_compiler_cache",
+      &clear_compiler_cache,
+      R"pbdoc(
+        Clear the compiler cache causing all compiled function to recompile if
+        they are called again.
+      )pbdoc");
```

Comment:
Do we need a binding for that / what is the intended use for it?

Reply:
I 'll remove it. I added it for testing and then thought that maybe someone wanted to clean things up for long running programs.